include/aos/common/crypto/mbedtls/driverwrapper.hpp

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (C) 2024 Renesas Electronics Corporation.
+ * Copyright (C) 2024 EPAM Systems, Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef DRIVERWRAPPER_HPP_
+#define DRIVERWRAPPER_HPP_
+
+#include <psa/crypto.h>
+
+#include "aos/common/crypto.hpp"
+
+/**
+ * @brief Add key to the list of builtin keys
+ *
+ * @param key Key description
+ *
+ * @return Error code
+ */
+aos::RetWithError<psa_key_id_t> AosPsaAddKey(const aos::crypto::PrivateKeyItf& mPrivKey);
+
+/**
+ * @brief Remove key from the list of builtin keys
+ *
+ * @param keyId Key ID
+ */
+void AosPsaRemoveKey(psa_key_id_t keyId);
```

Comment:
probably need to return Error for NotFound case

Reply:
I'm not sure this is necessary, and i think when we delete and the key is not present it's not an error